addons/hold-delete/addon.json

```diff
@@ -24,6 +24,26 @@
     }
   ],
   "settings": [
+    {
+      "name": "Hold duration",
+      "id": "duration",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "short",
+          "name": "Short"
+        },
+        {
+          "id": "default",
+          "name": "Default"
+        },
+        {
+          "id": "long",
+          "name": "Long"
+        }
+      ],
+      "default": "default"
+    },
     {
       "name": "Skip confirmation",
```

Comment:
Could this be a None option in the duration setting or was it left here so it's easier to find?

Reply:
What?

FYI, the hold duration setting controls how long you have to hold the button down to delete the sprite. There can't be a "none" option - you have to be able to hold it for some amount of time.

Were you talking about a different kind of delay?

Reply:
I meant for no holding, so you're right, keeping it this way is more intuitive.

Reply:
I just realized what this option is for and I don't think it makes sense - I thought the purpose of the addon was to replace the confirmation popup with the press and hold feature, not to effectively add a second confirmation. If we keep it, it should at least be on by default. And the delay setting should definitely have a "None" option to allow people to revert to Scratch's previous behavior.

Reply:
A "none" duration option would be equivalent to disabling the addon but just without the "skip confirmation" toggle, right?

If the addon says it adds a hold duration to the delete button, then why should it have an option for no hold duration? If the purpose of that option is to remove the confirmation, then we should rename the addon to indicate that it replaces the confirmation with something else so it makes more sense.

Reply:
I also thought this setting was for disabling the delay, not the confirmation prompt.

Reply:
I do agree we should reconsider the "skip confirmation" setting. Maybe we could instead update the addon description to say that the addon removes the confirmation and replaces it with a press and hold gesture, even though that feels like some people may miss that detail and then _oh no, the confirmation is gone!_

Reply:
And yes, the main reason I made this addon was so you could replace the confirmation with something else. Initially, though, I went with including a setting for whether or not to skip the confirmation after holding the delete button (off by default so users aren't surprised), so that's the origin of that setting.

Reply:
> I thought the purpose of the addon was to replace the confirmation popup with the press and hold feature

Is that not the purpose?

Reply:
> > I thought the purpose of the addon was to replace the confirmation popup with the press and hold feature
> 
> Is that not the purpose?

In the initial version of the PR, it didn't remove the vanilla confirmation by default.